File: uri-judge/[2930]TCC_Da_Depressao.java
//Author: Hrz - 2ºSM de CC
//Site: hrezend.github.io

import java.util.Scanner;

public class Main{
	public static void main(String[] args){
	
		Scanner hr = new Scanner(System.in);
		int E, D;
		E = hr.nextInt();
		D = hr.nextInt();
	
		if(E > D){
			System.out.println("Eu odeio a professora!");
		}
		else if(D-E > 2){
			System.out.println("Muito bem! Apresenta antes do Natal!");
		}
		else if(D-E < 3){
			System.out.println("Parece o trabalho do meu filho!");
			E+= 2;
			if(E >= 24){
				System.out.println("Fail! Entao eh nataaaaal!");
			}
			else{
				System.out.println("TCC Apresentado!");
			}
		}
	}
}

File: uri-judge/[2930]TCC_Da_Depressao.cpp
//Author: Hrz - 2ºSM de CC
//Site: hrezend.github.io

#include <bits/stdc++.h>
#define desync ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL)
#define endl '\n'
using namespace std;

int main(){

	desync;
	int E, D;
	cin >> E >> D;
	
	if(E > D){
		cout << "Eu odeio a professora!" << endl;
	}
	else if(D-E > 2){
		cout << "Muito bem! Apresenta antes do Natal!" << endl;
	}
	else if(D-E < 3){
		cout << "Parece o trabalho do meu filho!" << endl;
		E+= 2;
		if(E >= 24){
			cout << "Fail! Entao eh nataaaaal!" << endl;
		}
		else{
			cout << "TCC Apresentado!" << endl;
		}
	}
}
